snippets/product-variant-options.liquid

```diff
@@ -45,10 +47,25 @@
     endfor
   -%}
 
-  {%- if block.settings.picker_type == 'button' -%}
+  {%- liquid
+    if value.swatch.image
+      assign image_url = value.swatch.image | image_url: width: 50
+      assign swatch_value = 'url(' | append: image_url | append: ')'
+    elsif value.swatch.color
+      assign swatch_value = 'rgb(' | append: value.swatch.color.rgb | append: ')'
+    else
+      assign swatch_value = nil
+    endif
+  -%}
+
+  {%- capture input_id -%}
+    {{ section.id }}-{{ option.position }}-{{ forloop.index0 }}
+  {%- endcapture -%}
+
+  {%- capture radio_input -%}
```

Comment:
nit -- if radio_input is only used once below, do we need to wrap it in a `capture`?